src/qililab/qprogram/experiment_executor.py

```diff
@@ -553,11 +553,17 @@ def execute(self) -> str:
         # Create file path to store results
         results_path = self._create_results_path(executed_at=executed_at)
 
+        # TODO: Find an alternative way of getting the experiment path at the beginning of the experiment execution
+        if self.live_plot:
+            print(results_path)
```

Comment:
This print statement should be deleted?

Reply:
This print statement is the temporary "solution" to get the actual data required for the live plotting. The live plot currently works in a separate kernel (but I can also implement it to print at the same cell as it is being executed.

Reply:
A print statement cannot be a solution. In any case, it should be a warning or something else but not a print.

Reply:
Agreed with Juanjo, a `print()` here is not a good practice.

As the method returns the `results_path`, maybe let the caller print the variable themselves?